Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

In [1]:
std = 16
M = 80
a = 0.95
n = 256

Доверительный интервал 95% означает, что нам нужно будет использовать квантили 2.5% и 97.5%, которым соответствует значение из таблицы z-значений -1.96 и 1.96 соответственно.

Есть формула для доверительных интервалов: $\mu \pm z_{\alpha/2} \cdot \dfrac{\sigma}{\sqrt{n}}$

In [2]:
z = 1.96

Внимательно посмотрев на формулу, можно сказать, что для нахождения длины интервала потребуется сделать следующее:

$2 \cdot z_{\alpha/2} \cdot \dfrac{\sigma}{\sqrt{n}}$

Длина интервала никак не будет зависеть от выборочной средней.

In [3]:
from math import sqrt
i = 2 * z * (std / sqrt(n))
i

3.92

#### Ответ:
3.92

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
```
202, 203, 199, 197, 195, 201, 200, 204, 194, 190
```
Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что уровень значимости равен 1%? Найдите P-значение для данного теста и укажите его в качестве ответа.

In [14]:
import numpy as np
a = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
X = np.array(a)
alpha = 0.01
mu = 200
n = len(X)
std = X.std()
mean = X.mean()

In [13]:
X.std()

4.224926034855522

Уровню значимости 1% будут соответствовать квантилям 0.5% и 99.5%

Значения из z-таблицы для таких величин будут -2.58 и 2.58

Теперь найдём значение z по формуле: $z = \dfrac {\overline{X} - \mu}{\sigma / \sqrt{n}}$

In [15]:
z = (mean - mu) / (std/sqrt(n))

In [16]:
z

-1.1227217828476797

Видим, что расчётное значение $z$ входит в интервал определённый уровнем значимости.

Это значит, что продавец печенья был прав.

Найдём по таблице значение соответствующее z = -1.12

Это будет примерно 0.868

Теперь найдём P значние

In [17]:
b = 0.868

In [19]:
P = 2 * (1 - b)

0.264

#### Ответ:
Продавец был прав, P-значние = 0.264

С 3ей задачей не разобрался